include/industry_standard/spdm.h

```diff
@@ -365,6 +365,9 @@ typedef struct {
 #define SPDM_REGISTRY_ID_MIPI 6
 #define SPDM_REGISTRY_ID_CXL 7
 #define SPDM_REGISTRY_ID_JEDEC 8
+#define SPDM_REGISTRY_ID_VESA 9
+#define SPDM_REGISTRY_ID_IANA_CBOR 0xa
+#define SPDM_REGISTRY_ID_MAX  0xa
```

Comment:
This should be defined in `libspdm_common_lib.h` as `LIBSPDM_REGISTRY_ID_MAX`.

Reply:
Why? I think it should be in SPDM definition. Irrelevant to SPDM implementation.

Reply:
`spdm.h` should only reflect what's in the specification.

Reply:
Right. That is why we have below:
```
#define SPDM_MAX_VERSION_COUNT 3
#define SPDM_MAX_SLOT_COUNT 8
#define SPDM_MAX_OPAQUE_DATA_SIZE 1024
```
There is nothing wrong to add `SPDM_REGISTRY_ID_MAX` IMHO, because that is what the spec has defined.

Reply:
Ok.